### PR TITLE
add blender image file and add it to README

### PR DESCRIPTION
**Pull request by clmould**
## Description

Added blender generated image of reactor to the README
## Checklist

I confirm that I have completed the following checks:

- [x] I have justified any large differences in the regression tests caused by this pull request in the comments.
- [x] I have added new tests where appropriate for the changes I have made.
- [x] If I have had to change any existing unit or integration tests, I have justified this change in the pull request comments.
- [x] If I have made documentation changes, I have checked they render correctly.
- [x] I have added documentation for my change, if appropriate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ PROCESS is the reactor systems code at the [UK Atomic Energy Authority](https://
 
 PROCESS was originally a Fortran code, but is currently a mixture of Python and Python-wrapped Fortran; the eventual aim is to have an entirely Python code base. In order to use PROCESS, the Fortran must be compiled and a Python-Fortran interface generated for the Python to import. Once built, it can be installed and run as a Python package.
 
+![BlenderProcessImage](blenderimages/initial_collage.png)
```
Comment (jonmaddock) [inline README.md:8]: Minor: could this go into `images` instead or `blenderimages`? We might well have more non-blender images in future.

**Review by jonmaddock — COMMENTED** [review]
This is a good start Clair: this is just what we wanted. Could you check with Oli that these are the best images we have at the moment, and if there are any other views we should include (i.e. the side-by-side Process and Bluemira design view). If not, then I'm happy with this!

File: README.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ PROCESS is the reactor systems code at the [UK Atomic Energy Authority](https://
 
 PROCESS was originally a Fortran code, but is currently a mixture of Python and Python-wrapped Fortran; the eventual aim is to have an entirely Python code base. In order to use PROCESS, the Fortran must be compiled and a Python-Fortran interface generated for the Python to import. Once built, it can be installed and run as a Python package.
 
+
+<figure markdown>
+![BlenderProcessImage](documentation/proc-pages/images/blender images/initial_collage.png){ width="100%"}
+<figcaption>Split view of DEMO-like reactor (generated using fusrr-pipeline).</figcaption>
+</figure>
```
Comment (clmould) [inline README.md:12]: I could add the link to the fusrr repo when it is made public?

Comment (jonmaddock) [reply README.md:12]: Yeah, that's a good idea.

**Review by jonmaddock — APPROVED** [review]

**Pull request merged**